lib/TWCManager/Control/HTTPControl.py

```diff
@@ -754,6 +755,19 @@ def do_GET(self):
                 ).car_api_available()
                 self.scheduledAmpsMax = master.getScheduledAmpsMax()
 
+                policy = str(master.getModuleByName("Policy").active_policy)
+                if policy == "Charge Now":
+                    self.activeAction = 1
+                elif policy == "Scheduled Charging":
+                    self.activeAction = 1
+                elif policy == "Track Green Energy":
+                    self.activeAction = 3
+                elif policy == "Non Scheduled Charging":
+                    self.activeAction = master.settings.get("nonScheduledAction", 1)
+                else:
+                    logger.error(f"non-standard policy {policy!r}, cannot display charge action in webinterface")
```

Comment:
This will break horribly in the presence of custom policies.  I would suggest using the `policyIsGreen()` function in the policy module to determine if we're currently tracking green energy.  If not, switch on whether the value being offered is zero or non-zero.